### ISSUE TITLE
yolo v3 loss function

### ISSUE THREAD

**Issue by sachinruk** [pjreddie/darknet#1203]
Was wondering if anyone could provide some insights here. Fingers crossed the author will comment himself:
https://stats.stackexchange.com/questions/373266/yolo-v3-loss-function

**Comment by YunYang1994** [comment]
https://github.com/YunYang1994/tensorflow-yolov3 hope it helps you
